src/main/java/com/jfb/tickets/util/storage/MashSet.java

```diff
@@ -0,0 +1,41 @@
+package com.jfb.tickets.util.storage;
+
+import com.jfb.tickets.util.interfaces.Identifiable;
+import java.util.HashMap;
+
+public class MashSet<T> implements Identifiable {
+
+  private final int CLASS_ID = generateClassId();
+  static final Object PRESENT = new Object();
+
+  private HashMap<T, Object> internalMap;
```

Comment:
do not use HashMap, instead use array, ArrayList ot LinkedList

Reply:
Used LinkedList. Good practice.